src/mpl2/src/hier_rtlmp.cpp

```diff
@@ -801,7 +815,7 @@ void HierRTLMP::createBundledIOs()
     if (cluster_id == -1) {
       logger_->error(
           MPL,
-          2024,
+          1002,
```

Comment:
I doubt there are 1001 prior messages.  Please use consecutive values

Reply:
It was sequenced from 1000 -- reset it to 0